lib/sync-records.ts

```diff
@@ -26,8 +26,8 @@ export const syncWorkbookRecords = async ({
   spaceType: SpaceType;
 }) => {
   const accessToken = await getAccessToken({
-    clientId: process.env.ONBOARDING_CLIENT_ID as string,
-    secret: process.env.ONBOARDING_CLIENT_SECRET as string,
+    clientId: process.env.CLIENT_ID as string,
+    secret: process.env.CLIENT_SECRET as string,
   });
```

Comment:
These values can go back inside the function if they're the same everywhere, don't need to be parameters.